src/test/java/pl/rejmanbeata/bakery/controller/TestMapper.java

```diff
@@ -0,0 +1,13 @@
+package pl.rejmanbeata.bakery.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class TestMapper {
+    public static String asJsonString(final Object obj) {
+        try {
+            return new ObjectMapper().writeValueAsString(obj);
```

Comment:
as a small improvement and optimization, maybe we could have `new ObjctMapper` as a field so that it is created only once and we only reuse it:)?